Makefile.js

```diff
@@ -31,7 +31,6 @@ var NODE = 'node ',	// intentional extra space
 
 	// Utilities - intentional extra space at the end of each string
 	ISTANBUL = NODE + NODE_MODULES + 'istanbul/lib/cli.js ',
-	MOCHA = NODE_MODULES + 'mocha/bin/_mocha ',
```

Comment:
any reason to remove this? do we risk using the globally installed mocha instead in the api-test?